templates/layout/amp-iframe.html

```diff
@@ -9,7 +9,11 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
         {{{stylesheet '/assets/css/theme.css'}}}
         {{ getFontsCollection }}
-        <script src="{{cdn 'assets/modernizr-custom.js'}}"></script>
+
+        <script>
+            // Change document class from no-js to js so we can detect this in css
+            document.documentElement.className = 'js';
```

Comment:
what is this about?  seems a little hacky...  can we get the same result another way?

Reply:
This is the last remnant of removing Modernizr. Modernizr does essentially this same thing, and this `js` class name is used by css to present alternatives when Javascript is enabled or not.

Reply:
This is a quite common pattern. Not too sure about the way it is done, this is not `changing the document class from no-js to js` it is replacing whatever it is in the body with `js`.

It might be better if we are more granular with the change:

```
document.documentElement.classList.remove('no-js');
document.documentElement.classList.remove('js');
```

Reply:
Yeah that bugged me too when I was working in this code, @icatalina, might be outside the scope of this PR to worry about changing this but I do think that approach would be both more clear and safer.

Reply:
I'm not using `classList` here, but `className`, which is a string, and then using Javascript's String replace on it. Should be safe to use in all browsers. This is a pretty standard thing, and exactly what Modernizr is doing under the hood. 

https://github.com/Modernizr/Modernizr/blob/5eea7e2a213edc9e83a47b6414d0250468d83471/src/setClasses.js#L22

Reply:
Oh wait, I see -- I made a change to this in `base.html` but forgot to bring it into `amp-iframe.html`. I'll do that now.

Reply:
Yeah sorry to be clear I was talking about `className = 'js'`, not `className.replace`